python/cudf/cudf/core/column/column.py

```diff
@@ -282,7 +282,15 @@ def _concat(
 
         # Perform the actual concatenation
         if newsize > 0:
-            col = libcudf.concat.concat_columns(objs)
+            try:
+                col = libcudf.concat.concat_columns(objs)
+            except RuntimeError as e:
+                if "concatenated rows exceeds size_type range" in str(e):
```

Comment:
This error from libcudf is not specific to strings columns. The number of rows in any column cannot exceed `size_type`. This means concatenating 2 integer columns each with rows lengths greater than `size_type/2` would throw this error from libcudf.

The specific message for strings below only occurs for strings columns.

Reply:
I generalized the error to `"total size of output is too large for a cudf column"`, is that sufficient?